notebooks/clean.py

```diff
@@ -6,19 +6,22 @@
 
 BAD_CELL_METADATA_KEYS = { '_uuid', '_cell_guid', }
 
-def clean(nb, path):
-    fname = os.path.basename(path)
-    match = re.fullmatch(r'(tut|ex)_(\d+).ipynb', fname)
-    assert match, fname
-    ltmeta = dict(
-            lesson_index=int(match.group(2))-1,
-            type='tutorial' if match.group(1) == 'tut' else 'exercise',
-            )
-    nb['metadata']['learntools_metadata'] = ltmeta
+# TODO: this should probably be set on a per-track basis in track_config.yaml
+CLEAR_OUTPUT = 0
+
+def clean(nb_path):
+    nb = nbf.read(nb_path, 4)
+    clean_nb(nb)
+    nbf.write(nb, nb_path)
+
+def clean_nb(nb):
+    """Modify given NotebookNode in place to normalize/strip some extraneous metadata.
+    """
     nb['metadata']['language_info']['version'] = '3.6.5'
```

Comment:
Are we going to regret hardcoding this someday when 3.6.5 isn't accurate any more?

Reply:
We can always change it! AFAIK kernels doesn't care about this particular field. I just wanted to standardize on *some* value to avoid flip-flopping values polluting diffs. I would have set it to null if I could, but I believe that made jupyter unhappy, so I semi-arbitrarily chose 3.6.5 since it's what I have on my machine :)